examples/hello/Makefile

```diff
@@ -0,0 +1,19 @@
+# For illustrative and convenience purposes only; you don't have to use make,
```

Comment:
This seems a bit...weird...

Reply:
The comment, or the Makefile? I use the Makefile myself, which is why I decided to contribute it. But we can remove. I can also document how to create simple little aliases or bash functions to do the same thing as a memory aid.